job-service-acceptance-tests/src/test/java/com/hpe/caf/jobservice/acceptance/JobServiceDatabaseUtil.java

```diff
@@ -112,7 +113,7 @@ public static void assertJobRowDoesNotExist(final String jobId) throws SQLExcept
             PreparedStatement st = dbConnection.prepareStatement("SELECT * FROM job WHERE job_id = ?");
             st.setString(1, jobId);
             final ResultSet jobRS = st.executeQuery();
-            Assert.assertTrue(!jobRS.next());
+            Assert.assertTrue(!jobRS.next(), "Job "+jobId+" should not have been existed in job table");
```

Comment:
Reword this message, does not make sense.
```suggestion
            Assert.assertTrue(!jobRS.next(), "Job "+jobId+" should not exist in job table");
```